python/src/aiconfig/editor/server/server.py

```diff
@@ -811,9 +812,32 @@ def _op(
 
     signature: dict[str, Type[Any]] = {}
     return run_aiconfig_operation_with_request_json(
-        aiconfig, request_json, f"method_", _op, signature
+        aiconfig, request_json, f"method_{method_name}", _op, signature
     )
 
+@app.route("/api/delete_output", methods=["POST"])
+def delete_output() -> FlaskResponse:
+    """
+    Clears a single outputs in the server state's AIConfig based on prompt name.
+    """
+    method_name = MethodName("delete_output")
+    state = get_server_state(app)
+    aiconfig = state.aiconfig
+    request_json = request.get_json()
+
+    if aiconfig is None:
+        LOGGER.info("No AIConfig in memory, can't run clear outputs.")
+        return HttpResponseWithAIConfig(
+            message="No AIConfig in memory, can't run clear outputs.",
+            code=400,
+            aiconfig=None,
+        ).to_flask_format()
+
+    signature: dict[str, Type[Any]] = {"prompt_name": str}
+    operation = make_op_run_method(method_name)
```

Comment:
Oh cool, TIL that we already have existing API for this: https://github.com/lastmile-ai/aiconfig/blob/f0f052231feee5f15843e2d88a70f7b406527dd5/python/src/aiconfig/schema.py#L1005. Thought it was for deleting entire outputs only. We can probly clean this up and delete the line `existing_outputs = prompt.outputs`. cc @Ankush-lastmile do you remember why this line might have been needed?